tron/utils/scribereader.py

```diff
@@ -0,0 +1,158 @@
+import datetime
+import json
+import logging
+import operator
+import socket
+from functools import lru_cache
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+
+try:
+    from scribereader import scribereader  # type: ignore
+    from clog.readers import StreamTailerSetupError  # type: ignore
+except ImportError:
+    scribereader = None  # sorry folks, you'll need to add your own way to retrieve logs
+
+
+log = logging.getLogger(__name__)
+
+
+@lru_cache(maxsize=1)
+def get_region() -> str:
```

Comment:
nit: The Python junkie in me says turn these three habitat getters into one function and three `functools.partial`s, but it's not at all necessary.

Reply:
as in something like this?
```Python
from functools import partial
def read_file(path: str) -> str:
    with open(path) as f:
        return f.read().strip()

get_region = partial(read_file, "/nail/etc/region")
...
```

Reply:
yeah :P but again im just being picky